src/GitVersion.Core/Configuration/GitFlowConfigurationBuilder.cs

```diff
@@ -133,6 +134,7 @@ private GitFlowConfigurationBuilder()
                 HotfixBranch.Name
             },
             Label = "beta",
+            IsReleaseBranch = true,
```

Comment:
So `hotfix` is considered a release branch now? We should probably document that.

Reply:
Yep I agree. I can add a point to the breaking change documenatation no problem.